src/tests/ftest/container/SnapshotInfo.py

```diff
@@ -0,0 +1,550 @@
+                self.log.info("==After 10 additional commits the snapshot is "
+                   "still available")
+            except Exception as error:
+                self.fail("##Test was unable to list the snapshot\n{0}"
+                   .format(error))
+
+        #(5)Verify the snapshots data
+        try:
+            for ind in range(0, len(self.container_epoch_list)):
+                epoch = self.container_epoch_list[ind]
+                ss = self.snapshot_list[ind]
+                datasize = len(self.test_data[ind]) + 1
+                obj.open()
+                snap_handle = self.snapshot.open(self.container.coh, ss.epoch)
+                thedata3 = self.container.read_an_obj(datasize, dkey, akey, obj,
+                                                  snap_handle.value)
+                self.log.info("==(5)snapshot test list {0}:".format(ind+1))
+                self.log.info("==self.container_epoch_list[ind]={0}"\
+                    .format(epoch))
+                self.log.info("==self.snapshot_list[ind]={0}"\
+                    .format(self.snapshot_list[ind]))
+                self.log.info("==self.snapshot_list[ind].epoch={0}"\
+                    .format( ss.epoch))
+                self.log.info("==self.test_data_size=    {0}".format(datasize))
+                self.log.info("==thedata3.value=         {0}"\
+                    .format(thedata3.value))
+                self.log.info("==self.test_data[ind]=    {0}"\
+                    .format( self.test_data[ind]))
+                #self.cancel("tickets already assigned DAOS-2484")
+                #if thedata3.value != self.test_data[ind]:
+                #    raise Exception("##The data in the snapshot is not "
+                #                    "same as the original data")
+                #self.log.info("The snapshot data matches the data originally "
+                #    "written.")
+        except Exception as error:
+            self.fail("Error when retrieving the snapshot data.\n{0}"
+                .format(error))
+
+        #(6)Destroy the snapshot
+        self.log.info("==(6)Destroy snapshot  epoch:{0}".format(epoch))
+        try:
+            self.snapshot.destroy(self.container.coh, epoch)
+            self.log.info("==Snapshot successfully destroyed")
+        except Exception as error:
+                self.fail("##{0}".format(error))
+
+        #(7)Check if still able to Open the destroied snapshot and
+        #    Verify the snapshot removed from the snapshot list
+        try:
+            obj.open()
+            snap_handle3 = self.snapshot.open(self.container.coh,
+                self.snapshot.epoch)
+            thedata3 = self.container.read_an_obj(datasize, dkey, akey,
+                obj, snap_handle3.value)
+            self.log.info("-->thedata_after_snapshot.destroied.value= {0}"\
+                .format(thedata3.value))
+            self.log.info("==>snapshot_epoch=     {0}"\
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments